frontend/src/components/mail-verification/verification-success/success.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+import Logo from '@assets/images/logo.svg';
+import { AppRoute } from '@common/enums/app/app';
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import Container from '@mui/material/Container';
+
+import styles from './styles.module.scss';
+
+const Success: FC = (): React.ReactElement => {
```

Comment:
`Success` -> `EmailVerificationSucess`

Reply:
### **Also, **_very important_**, the React components files are named from the capital letter!**